src/components/templates/protected-area/credentialsPage.tsx

```diff
@@ -1,11 +1,19 @@
 import React from "react"
 import { useServiceContext } from "../../../hooks/useServiceContext"
+import Password from "./password"
 
 function CredentialsPage() {
   const Services = useServiceContext({ fetchOnLoad: true })
 
   console.log("services", Services)
-  return <p>Protected area</p>
+  return (
+    <Password
+      history={["h!story1", "h!story2", "h!story3"]}
+      actualPassword="myActualPassw123!!"
+      newPassword="myN3wAmaz1ngPassw!!"
+      strength="81%"
```

Comment:
It would be better if `strength` was a `double` between `0` and `1`

Also, I think `strength` should be evaluated inside the component itself instead of being passed externally